examples/resources/main.tf

```diff
@@ -0,0 +1,3 @@
+resource "aws_cloudwatch_log_group" "test" {
```

Comment:
You'll want to include a reference to the local module so it will look something like
```
module "cloudtrail_alarms" {
  source = "../../"
  alarm_sns_topic_arn = aws_sns_topic.my_alerts.arn
}
```